pkg/bsp-imx/Dockerfile

```diff
@@ -0,0 +1,74 @@
+# syntax=docker/dockerfile-upstream:master-labs
+FROM lfedge/eve-alpine:8e3ddd8016f778932e9c204a3ec195ed74d1600a as build-base
+RUN mkdir /bsp
+
+FROM build-base AS build-amd64
+FROM build-base AS build-riscv64
+FROM build-base AS build-arm64
+ENV BUILD_PKGS bash binutils-dev build-base bc bison flex openssl-dev util-linux-dev swig gnutls-dev perl python3 python3-dev py3-setuptools py3-pycryptodome py3-elftools
+RUN eve-alpine-deploy.sh
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+
+ENV UBOOT_TARGETS "imx8mq_evk"
+ENV ATF_TARGETS "imx8mq"
+
+RUN mkdir /uboot-firmwares
+
+# ATF
+WORKDIR /imx-atf
+# last commit from imx_5.4.70_2.3.0 branch
+ENV ATF_COMMIT 15e8ff164a8becfddb76cba2c68eeeae684cb398
+ADD https://github.com/nxp-imx/imx-atf.git#${ATF_COMMIT} .
+RUN for target in ${ATF_TARGETS}; do \
+        make clean && make -j "$(getconf _NPROCESSORS_ONLN)" PLAT=$target bl31 ;\
+        mv build/imx8mq/release/bl31.bin /uboot-firmwares/${target}-bl31.bin ;\
+    done
+
+# IMX firmware
+WORKDIR /tmp
+ENV FIRMWARE_VER=8.18
+ADD https://www.nxp.com/lgfiles/NMG/MAD/YOCTO/firmware-imx-${FIRMWARE_VER}.bin firmware-imx-${FIRMWARE_VER}.bin
+RUN chmod 777 firmware-imx-${FIRMWARE_VER}.bin && \
+        ./firmware-imx-${FIRMWARE_VER}.bin --auto-accept && \
```

Comment:
by using --auto-accept here you are enforcing the user to agree with FSL EULA (https://github.com/Freescale/meta-fsl-arm/blob/master/EULA ?), in the imx BSPs usually the user must add by himself the accept variable in his yocto's local.conf file. @eriknordmark , would that be a problem for us? The point here is that this firmware package prompts the FSL EULA license to the user that must agree before get files extracted. But I don't think we are able to do the same here (due to the container output)...

Reply:
I suggested that we can neglect this, since we do not give out these files and do not store them. We only store the finished boot blob. As with the download of the AdvRaspbian operating system (imx dist), the user does not agree to anything when downloading or installing it, but in the image of the AdvRaspbian image there is approximately the same blob.

Reply:
I took a quick look on the license. You are right by saying that we don't store any binary on EVE sources. However, when generating the image for i.MX platform the binaries will be fetched and will be present in the final image. According to the license terms, item 3.4:

`3.4.             You must reproduce any and all of NXP's (or its third party
licensor's) copyright notices and other proprietary legends on copies of
Licensed Software.`

Include a copy of the license in the final image should be sufficiency to meet this requirement, and explains why you saw the EULA.txt in the tarball.

Reply:
> able to do the same here (due to the container output)

As I understand the main problem here is how to show the licence to the end-user of lfedge/eve image, correct? Probably we should add another option to runme.sh of lfedge/eve to show (i.e. `--show-licence`) and accept (i.e. `--accept-licence`) licence? We can disable extraction of image in case of target `imx8mq_evk` provided without `--accept-licence` flag.

Reply:
@giggsoff thanks for idea, implemented

For now: 
1) make live - creates live.raw and NXP-EULA-LICENSE.txt in the dist directory.
2) docker run - checks the --accept-license flag if the imx8mq_evk platform is selected.